File: trail2/숫자로 이루어진 사각형/rectangle-with-a-number.cpp
#include <iostream>
using namespace std;

void printSquare(int n) {
    int num = 1;

    for (int i = 0; i < n; i++) {
        for (int j = 0; j < n; j++) {
            if (num > 9) num = 1;

            cout << num << " ";
            num++;
        }
        cout << endl;
    }
}

int main() {
    // Please write your code here.
    int n;
    cin >> n;

    printSquare(n);

    return 0;
}

File: trail2/숫자로 이루어진 사각형/rectangle-with-a-number.py
n = int(input())

# Please write your code here.
def print_rect(num):
    cnt = 1
    for _ in range(num):
        for _ in range(num):
            print(cnt, end=" ")
            cnt += 1
            if cnt == 10:
                cnt = 1
        print()

print_rect(n)
